src/fontr/pipelines/data_science/nodes.py

```diff
@@ -218,9 +222,10 @@ def evaluate_classifier(
         recall.update(preds, y)
         accuracy.update(preds, y)
 
-    logging.info(f"Precision: {precision.compute():0.3f}")
```

Comment:
Can you add a TODO to add logging of MSE to `evaluate_autoencoder` as in here?

Can we log the metrics while training both models? What do you think?